### DISCUSSION TITLE
Addl → Additional [4.0.0]

### DISCUSSION THREAD

**Discussion by bencroker** [craftcms/cms#10741] [Ideas] [answered]
There are several instances in the source code where `addl` is used in place of `additional`. I get that it saves characters, but it feels slightly out of character and is not particularly intuitive. Some examples:

https://github.com/craftcms/cms/blob/6097d92a3710febb8a0cea17f7f69bd4ee949dba/src/base/Element.php#L2849-L2855

https://github.com/craftcms/cms/blob/6097d92a3710febb8a0cea17f7f69bd4ee949dba/src/controllers/ElementsController.php#L689

https://github.com/craftcms/cms/blob/6097d92a3710febb8a0cea17f7f69bd4ee949dba/src/controllers/ElementsController.php#L393

https://github.com/craftcms/cms/blob/6097d92a3710febb8a0cea17f7f69bd4ee949dba/src/web/CpScreenResponseBehavior.php#L391-L396

I understand that the effort may not be worth the trouble, but perhaps in the long run  it would be, and 4.0.0 may be a good opportunity to rid the codebase of this atrocity (I kid).

**Comment by brandonkelly** [comment, marked as answer]
That was just added in Craft 4. Renamed everything to use `additional` for the next release (764119f96e8d5ee597fe36ba370e385da3f94b06).

**Reply by brandonkelly** [reply]
Craft 4.0.0-beta.2 is out now with those changes.
